.github/workflows/pr.yml

```
@@ -205,13 +205,14 @@ jobs:
   analyze:
     name: Analyze Build
     needs: [build]
+    if: ${{ needs.check-label.outputs.run-e2e == 'true' && needs.changes.outputs.has-files-requiring-all-checks == 'true' }}
```

actionlint (1.7.7)

property "check-label" is not defined in object type {build: {outputs: {}; result: string}}

(expression)